css-two-column-layout/styles.css

```diff
@@ -0,0 +1,70 @@
+body {
+  font-family: "Karla", sans-serif;
+  line-height: 1.75rem;
+}
+
+h1 {
+  font-size: 40pt;
+  font-weight: bolder;
+}
+
+img {
+  /* height: 100%; */
+}
+
+.container {
+  max-width: 1000px;
+  width: 100%;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 60px;
+}
+
+.row {
+  display: flex;
+}
+
+.col-full {
+  flex-basis: 100%;
+  display: flex;
+  justify-content: center;
```

Comment:
If you are going to put `display: flex` on a column class, that is a clear indicator that it should have a `row` class inside of it. Column classes should only ever have a `flex-basis` or `width` associated with them